python/ray/_private/structured_logging.py

```diff
@@ -0,0 +1,42 @@
+import logging
```

Comment:
Let's put this under `python/ray/_private/logging/` module

Reply:
Do you mean to create a new directory named `logging/`?

Reply:
I create a new directory `logging/` under `ray/_private` https://github.com/ray-project/ray/pull/45098/commits/b2b2196a31eb8a907c4ca73038acfe7ee1a33de4. Let me know if my understanding is incorrect.